Frontend/src/main.js

```diff
@@ -10,3 +10,18 @@ new Vue({
   router,
   render: h => h(App)
 }).$mount('#app')
+
+// Add a request interceptor
+axios.interceptors.request.use(function (config) {
+  return config;
+}, function (error) {
+  return Promise.reject(error);
+});
+
+// Add a response interceptor
+axios.interceptors.response.use(function (response) {
+  return response;
+}, function (error) {
+  alert(error.response.data);
+  return Promise.reject(error);
+});
```

Comment:
We need to remove this - it prints a log every time a request is posted. I wanted to merge this before the demo so it can be deployed - ping me on Discord when you can.